src/java/edu/slu/action/ObjectAction.java

```diff
@@ -740,6 +768,16 @@ public void saveNewObject() throws IOException, ServletException, Exception{
             //set @id from _id and update the annotation
             BasicDBObject dboWithObjectID = new BasicDBObject((BasicDBObject)dbo);
             String uid = "http://devstore.rerum.io/rerumserver/id/"+newObjectID;
+            if(dboWithObjectID.containsKey("@id")){
+                //Someone tried to create an object that already had an @id.  How do we want to handle this?
```

Comment:
I thought we were rejecting create actions with @id. I think we need to throw a method error with a note to Update objects with @ids.

Reply:
done deal